getdeps.sh

```diff
@@ -62,14 +62,31 @@ function build_cmake() {
     )
 }
 
+function build_gtest_and_gmock() {
+    (
+        for LIB in gmock gtest
+        do
+          echo "building $LIB..."
+          cd "/usr/src/$LIB" || die "Missing dir /usr/src/$LIB"
+          if [ -e ./CMakeLists.txt ]; then
+              echo cmake .. $CMAKEFLAGS
+              sudo cmake CMakeLists.txt $CMAKEFLAGS || die "Cmake failed for $LIB"
+              sudo make -j "$NPROC" || die "Failed to build $LIB"
+          else
+        die "No CmakeLists.txt found for $LIB"
+          fi
+        done
+    )
+}
+
 function get_packages() {
     echo "installing packages"
     sudo apt-get install -yq autoconf automake libdouble-conversion-dev \
         libssl-dev make zip git autoconf libtool g++ libboost-all-dev \
         libevent-dev flex bison libgoogle-glog-dev scons libkrb5-dev \
         libsnappy-dev libsasl2-dev libnuma-dev libi2c-dev libcurl4-nss-dev \
         libusb-1.0-0-dev libpcap-dev libdb5.3-dev cmake libnl-3-dev \
-        libnl-route-3-dev iptables-dev
+        libnl-route-3-dev iptables-dev libgtest-dev googlemock
```

Comment:
On my system this is 'google-mock' (with a hyphen) - is this a typo?

Reply:
fixed